awx/main/analytics/collectors.py

```diff
@@ -166,7 +172,7 @@ def instance_info(since):
             'cpu': instance['cpu'],
             'memory': instance['memory'],
             'managed_by_policy': instance['managed_by_policy'],
-            'last_isolated_check': instance['last_isolated_check'],
+            'last_isolated_check': _get_isolated_datetime(instance['last_isolated_check']),
```

Comment:
so if there is no isolated node what happens?
this method returns `None` right?
is that ok?

Reply:
Yep - this is going to be serialized to JSON as `null`.